src/AzureFunctions.Extensions.Swashbuckle/AzureFunctions.Extensions.Swashbuckle/FunctionApiDescriptionProvider.cs

```diff
@@ -42,14 +42,16 @@ internal class FunctionApiDescriptionProvider : IApiDescriptionGroupCollectionPr
                 .Where(m => m.GetCustomAttributes(typeof(FunctionNameAttribute), false).Any())
                 .ToArray();
 
-            IList<ApiDescriptionGroup> apiDescrGroup = new List<ApiDescriptionGroup>();
+            var apiDescGroups = new Dictionary<string, List<ApiDescription>>();
             foreach (var methodInfo in methods)
             {
                 if (!TryGetHttpTrigger(methodInfo, out var triggerAttribute))
                     continue;
 
                 var functionAttr =
                     (FunctionNameAttribute)methodInfo.GetCustomAttribute(typeof(FunctionNameAttribute), false);
+                var apiExplorerSettingsAttribute = 
```

Comment:
I want to conside when ApiExplorerSettingAttribute is attached to the class to which FunctionMethod belongs.

In that case, I think that you should change like this.

```
var apiExplorerSettingsAttribute = 
                    (ApiExplorerSettingsAttribute)methodInfo.GetCustomAttribute(typeof(ApiExplorerSettingsAttribute), false) ??
                    (ApiExplorerSettingsAttribute)methodInfo.DeclaringType.GetCustomAttribute(typeof(ApiExplorerSettingsAttribute), false);
```

What do you think about this opinion?

Reply:
Yes, good idea!